<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_Preview"/>
    <addaction name="actionPDF"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuStressed">
    <property name="title">
     <string>Stressed...</string>
    </property>
    <addaction name="actionPlay"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionMenu"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="separator"/>
    <addaction name="actionPos_Left"/>
    <addaction name="actionPos_Right"/>
    <addaction name="actionPos_Center"/>
    <addaction name="actionJustify"/>
    <addaction name="actionColor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuStressed"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/filenew.png</normaloff>:/Images/images/win/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/fileopen.png</normaloff>:/Images/images/win/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/filesave.png</normaloff>:/Images/images/win/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as... </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/fileprint.png</normaloff>:/Images/images/win/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>Print </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionPDF">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/exportpdf.png</normaloff>:/Images/images/win/exportpdf.png</iconset>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/editundo.png</normaloff>:/Images/images/win/editundo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/editredo.png</normaloff>:/Images/images/win/editredo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/editcopy.png</normaloff>:/Images/images/win/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/editpaste.png</normaloff>:/Images/images/win/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/editcut.png</normaloff>:/Images/images/win/editcut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/mona.png</normaloff>:/Images/images/win/mona.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionMenu">
   <property name="text">
    <string>Menu</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/textbold.png</normaloff>:/Images/images/win/textbold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/textitalic.png</normaloff>:/Images/images/win/textitalic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/textunder.png</normaloff>:/Images/images/win/textunder.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionPos_Left">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/textleft.png</normaloff>:/Images/images/win/textleft.png</iconset>
   </property>
   <property name="text">
    <string>Pos Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionPos_Right">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/textright.png</normaloff>:/Images/images/win/textright.png</iconset>
   </property>
   <property name="text">
    <string>Pos Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionPos_Center">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/textcenter.png</normaloff>:/Images/images/win/textcenter.png</iconset>
   </property>
   <property name="text">
    <string>Pos Center</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionJustify">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Images/images/win/textjustify.png</normaloff>:/Images/images/win/textjustify.png</iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
